src/nodes.js

```diff
@@ -596,7 +597,7 @@ module.exports = {
   var_ref: first,
   vcall: first,
   when: (path, opts, print) => {
-    const [_predicates, _statements, addition] = path.getValue().body;
```

Comment:
For what it's worth, these unused variables have been useful to me when debugging the AST. Particularly when we implicitly make use of these values later when calling things like `path.call(print, "body", 1)`. Maybe we should turn off this linting rule to help future developers 🤔